pygluu/kubernetes/gui/install.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+import threading
+import time
+import os
+import subprocess
+import shlex
+
+from math import sqrt
+from flask import Blueprint, render_template, current_app
+from flask_socketio import emit
+from pygtail import Pygtail
+from queue import Queue
+
+from pygluu.kubernetes.kustomize import Kustomize
+from pygluu.kubernetes.helm import Helm
+from pygluu.kubernetes.settingdb import SettingDB
+from .extensions import socketio
+import sys
+install = Blueprint('install', __name__, template_folder="templates")
+settings = SettingDB()
+timeout = 120
+queue = Queue()
+
+
+
+@install.route('/install')
+def install_kustomize():
+    t = threading.Thread(target=do_installation, args=(queue,))
+    t.daemon = True
+    t.start()
+    return render_template('install/kustomize.html')
+
+
+@socketio.on('install', namespace='/test')
+def installer_logs():
+    data = {}
+    tail_log = True
+    while tail_log:
+        if not queue.empty():
+            data = queue.get()
+
+        logs = Pygtail("./setup.log", read_from_end=True, every_n=1)
```

Comment:
Logs  should always be at the last line but it should be able to allow user to scroll and view the logs from the beginning.